src/bedrock/monitor/feedback_handler.cpp

```diff
@@ -4,7 +4,7 @@ void feedback_handler(
     zmq::socket_t* feedback_puller,
     std::unordered_map<std::string, double>& user_latency,
     std::unordered_map<std::string, double>& user_throughput,
-    std::unordered_map<Key, std::pair<double, unsigned>>& rep_factor_map) {
+    std::unordered_map<Key, std::pair<double, unsigned>>& bump_factor_map) {
```

Comment:
Can we instead call this the `latency_miss_ratio_map`?

Reply:
fixed